docs/jade-getting-started.md

```diff
@@ -366,7 +366,7 @@ export GOOGLE_ALLOWREUSEEXISTINGPROJECTS=true
 # Setting for credentials to test on Azure
 export AZURE_CREDENTIALS_HOMETENANTID=<value from the top of .github/workflows/int-and-connected-test-run.yml>
 export AZURE_CREDENTIALS_APPLICATIONID=<value from the top of .github/workflows/int-and-connected-test-run.yml>
-export AZURE_CREDENTIALS_SECRET=$(vault read -format=json secret/dsde/datarepo/dev/helm-azure | jq -r .data.applicationsecret)
+export AZURE_CREDENTIALS_SECRET=$(cat /tmp/jade-dev-azure.key)
```

Comment:
Would it be possible/useful to make `render-configs.sh` generate this block of `exports` as a `sh` script in `/tmp/`?

Reply:
@nmalfroy suggested we source `render-configs.sh`, but I hesitate to make any additional changes at this time.